src/CMakeLists.txt

```diff
@@ -769,7 +769,7 @@ if (NOT WITH_SYSTEM_ROCKSDB)
   list(APPEND ROCKSDB_CMAKE_ARGS -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE})
 
   if (CMAKE_CXX_COMPILER_ID STREQUAL Clang)
-    list(APPEND ROCKSDB_CMAKE_ARGS -DFAIL_ON_WARNINGS=OFF)
+    list(APPEND ROCKSDB_CMAKE_ARGS -DFAIL_ON_WARNINGS=OFF -Wnoexpansion-to-defined)
```

Comment:
@tchaikov 
Is does not get set indeed ... So my fix was bogus.
And now I wonder WHY i had these errors in the first place?

I think I had them during a new portversion build, and then I did move rocksdb forward there to include the recent patches, I guess that fixed the warnings.
Running the port build again